pkg/healthcheck/microshift_core_workloads.go

```diff
@@ -30,15 +32,76 @@ func getCoreMicroShiftWorkloads() (map[string]NamespaceWorkloads, error) {
 			},
 		},
 	}
-	fillOptionalWorkloadsIfApplicable(cfg, workloads)
+	if cfg.Network.IsEnabled() {
+		workloads["openshift-ovn-kubernetes"] = NamespaceWorkloads{
+			DaemonSets: []string{"ovnkube-master", "ovnkube-node"},
+		}
+	}
+	if err := fillOptionalWorkloadsIfApplicable(cfg, workloads); err != nil {
+		return nil, err
+	}
 
 	return workloads, nil
 }
 
-func fillOptionalWorkloadsIfApplicable(cfg *config.Config, workloads map[string]NamespaceWorkloads) {
-	klog.V(2).Infof("Configured storage driver value: %q", string(cfg.Storage.Driver))
-	if cfg.Storage.IsEnabled() {
-		klog.Infof("LVMS is enabled")
+func lvmsIsExpected(cfg *config.Config) (bool, error) {
+	cfgFile := "/etc/microshift/lvmd.yaml"
+	if exists, err := util.PathExists(cfgFile); err != nil {
+		return false, err
+	} else if exists {
+		klog.Infof("%s exists - expecting LVMS to be deployed", cfgFile)
+		return true, nil
+	}
+
+	if !cfg.Storage.IsEnabled() {
+		klog.Infof("LVMS is disabled via config. Configured value: %q", string(cfg.Storage.Driver))
+		return false, nil
+	}
+
+	cmd := exec.Command("vgs", "--readonly", "--options=name", "--reportformat=json")
+	output, err := cmd.Output()
+	if err != nil {
+		return false, err
+	}
+	klog.V(2).Infof("vgs reported: %s", strings.ReplaceAll(strings.ReplaceAll(string(output), "\n", ""), " ", ""))
+
+	report := struct {
+		Report []struct {
+			VGs []struct {
+				VGName string `json:"vg_name"`
+			} `json:"vg"`
+		} `json:"report"`
+	}{}
+
+	err = json.Unmarshal(output, &report)
+	if err != nil {
+		return false, err
+	}
+
+	if len(report.Report) == 0 || len(report.Report[0].VGs) == 0 {
+		klog.Infof("Detected 0 volume groups - LVMS is not expected")
+		return false, nil
+	}
+
+	if len(report.Report[0].VGs) == 1 {
+		klog.Infof("Detected 1 volume groups - LVMS is expected")
```

Comment:
Could we print the name of the detected group?

Reply:
It's in the `vgs reported` log